inz-demo/internetnz/Issues/nethuiTicket/description.xml

```diff
@@ -0,0 +1,123 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo INZ Nethui Ticket</en>
+		<nl>Demo INZ Nethui Ticket</nl>
+	</Name>
+	<ShortName>
+		<en>Demo INZ Nethui Ticket</en>
+		<nl></nl>
+	</ShortName>
+	<SchemeManager>inz-demo</SchemeManager>
+	<IssuerID>internetnz</IssuerID>
+	<CredentialID>nethuiTicket</CredentialID>
+	<Description>
+		<en>INZ Nethui Ticket</en>
+		<nl></nl>
+	</Description>
+	<IssueURL>
+		<en></en>
+		<nl></nl>
+	</IssueURL>
+	<Attributes>
+		<Attribute id="id">
+			<Name>
+				<en>Ticket number</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Ticket number</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="event">
+			<Name>
+				<en>Event</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Event</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="eventDate">
+			<Name>
+				<en>Event date</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Event date</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="ticketType">
+			<Name>
+				<en>Ticket type</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Ticket type</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="fullName">
+			<Name>
+				<en>Full name</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Full name</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="email">
+			<Name>
+				<en>E-mail</en>
```

Comment:
Do we need an email if we used an email attribute already on the phone?

Reply:
Same as for the name. I believe this is a more relaxed credential. The buyer could be different from the ticket user.

Reply:
I dont think someone would store someone else's ticket on their digital identity wallet as such given the use case for these wallets is peoples personal attributes and digital identity related attributes. We have to be careful here not to be duplicating other ticket services like eventbrite etc.

Reply:
If you bought a ticket for someone else, I'd think that it either emails that other person the ticket or that person gets sent the QR code to load into their own IRMA wallet.

Reply:
Let's keep this simple, InternetNZ member can prove they are a member to get the discount for their access pass. Their access pass can be issued to their IRMA wallet proving they are allowed to attend the series of online and in-person nethui events (the user could buy online, in-person or both as ticket types). Should we then also build a simple online access website and a kiosk webpage to allow people to "attend" the events to finish off a nice overall PoC flow?

Reply:
Let's not worry about issuing other's tickets for this PoC, we can just fake that bit.

Reply:
OK. I removed fullName and email.

A page to "attend" is good! Will do that.